### ISSUE TITLE
Need to refactor JWT auth

### ISSUE THREAD

**Issue by tarlepp** [tarlepp/angular-sailsjs-boilerplate#66]
And why? That's why https://auth0.com/blog/2015/03/31/critical-vulnerabilities-in-json-web-token-libraries/

**Comment by eddieajau** [comment]
So is your current implementation vulnerable or is it safe to use? Thanks.

**Comment by tarlepp** [comment]
Haven't tested this, but I'm going to take a deeper look for this. Also I'm going to change that JWT part bit; basically add max age of JWT.

**Comment by eddieajau** [comment]
Looks like there is a critical update for `jsonwebtoken` in the backend. Let me know if you need any help.

**Comment by LeonardoGentile** [comment]
Hi, thank you very much for this project, this saves me a huge amount of setup time!
About this issue, is it related only to the libraries you are using ? In this case have you already updated the package.json to included the patched version? 
Or is it coming from your implmentation you make of jwt?

Thanks

**Comment by tarlepp** [comment]
@LeonardoGentile Yeah, that's the plan but I really don't know when I have time for this. You guys should update ```jsonwebtoken``` to latest version.
